source/nodejs/adaptivecards-ui-testapp/src/rendering-utils.ts

```diff
@@ -89,6 +87,16 @@ export function renderCard(cardJson: any, callbackFunction: Function): void {
                 inputsMap[input.id] = input.value;
             });
 
+            if (actionType === SubmitAction.JsonTypeName)
+            {
+                const submitAction: SubmitAction = action as SubmitAction;
```

Comment:
nit: no need for an explicit type if you're already using a type assertion...

```suggestion
                const submitAction = action as SubmitAction;
```